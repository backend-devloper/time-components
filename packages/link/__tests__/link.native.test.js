import test from "./link-helper";
import Link from "../link.js";

test(Link);
