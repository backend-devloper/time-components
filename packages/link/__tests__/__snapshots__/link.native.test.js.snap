// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with children 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  onPress={[Function]}
  style={
    Array [
      Object {
        "color": "#069",
      },
      Object {},
    ]
  }
>
  The Times
</Text>
`;

exports[`renders correctly with specifc styles 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  onPress={[Function]}
  style={
    Array [
      Object {
        "color": "#069",
      },
      Object {
        "backgroundColor": "blue",
      },
    ]
  }
>
  The Times
</Text>
`;
